release_linux_client.sh

```diff
@@ -76,13 +78,8 @@ done
 # Update the stable download, i.e. that linked from getoutline.org.
 cp Outline-Client.AppImage stable/
 
-# Just the version number, e.g.:
-#   linux-v1.0.3 -> v1.0.3
-readonly VERSION=$(echo "${TAG}" | cut -d'-' -f2)
-
-git checkout -b linux-client-"${VERSION}"
-git commit -a -m "release linux client ${VERSION}"
+git commit -a -m "release linux client v${SEMVER_VERSION}"
```

Comment:
I'm of two minds on committing straight to master. One, the `release candidate -> release` step is -effectively- a code review so it feels like doing an additional review here is now redundant and weird, especially since we're triggering it after the release has been made on the client-side.

I'm not too worried about spurious pushes to master, as we have branch guards, etc. However, it would still be nice to get a review in if someone needed to run this script manually for whatever reason. Perhaps this now lives in the client repository?